facegame/common/static/js/fgscript.js

```diff
@@ -134,6 +134,8 @@ function initialize() {
     return false;
 
   });
+
+  $(".siteChoice").attr("checked", true);
```

Comment:
The JS quote style should be unified to either single quotes everywhere or double quotes everywhere, now it's a mixed bag.